text/0232-maturity-of-otel.md

```diff
@@ -0,0 +1,153 @@
+
+#### Deprecated
+
+Development of this component is halted, and no further support will be provided. No new versions are planned, and the component might be removed from its included distributions. Note that new issues will likely not be worked on except for critical security issues. Components that are included in distributions are expected to exist for at least two minor releases or six months, whichever happens later. They also MUST communicate in which version they will be removed.
+
+#### Unmaintained
+
+A component identified as unmaintained does not have an active code owner. Such components may have never been assigned a code owner, or a previously active code owner has not responded to requests for feedback within 6 weeks of being contacted. Issues and pull requests for unmaintained components SHOULD be labeled as such. After 6 months of being unmaintained, these components MAY be deprecated. Unmaintained components are actively seeking contributors to become code owners.
+
+### Specification requirements
+
+The specification should declare all requirements for a specified component to be declared stable. When implementations are still missing features required by the specification, they can be at most Alpha status.
+
+### Tier-1 SIGs
+
+This OTEP recommends the following SIGs to be included as part of the Tier-1 SIGs for OpenTelemetry:
+
+* Spec General
+* Semantic Conventions (WG)
+* Collector
+* Java
+* Go
+* .NET
+* JavaScript
+
+Once this OTEP is approved, the TC and GC members will vote on the individual SIGs from the list above and are free to make further suggestions. A final list of Tier-1 SIGs shall be in the "community" repository.
+
+### Tier-1 Components
+
+When discussing the components' maturity, we often debate what should be included in the evaluation criteria. To help clarify that, we introduce the concept of "Tier-1 SIGs" and "Tier-1 Components". Tier-1 Components are critical for a specific SIG: only when all Tier-1 Components are stable can a project be evaluated as stable. Example:
+
+* The "OTLP receiver" for the OpenTelemetry Collector could potentially be a "Tier-1 Component". Only after this component is declared stable may the Collector be declared stable, provided all other Tier-1 Components are also stable.
+
+Projects MUST list their Tier-1 Components in the main readme file. A Tier-1 Component MAY be hosted under the project's "contrib" repository. Not all components under the main repository (sometimes called the "core" repository) are automatically Tier-1 Components.
+
+A component can be declared "Tier-1" only if they have the following characteristics:
+
+* The component has at least the same maturity level as the SIG
+* The component has a clear code owner
+* The code owner is active within the project
+* The maintainers of the project are comfortable supporting the component should the code owner decide to step down or leave the project
+
+### Tier-1 Signals
```

Comment:
I feel like this should just be calling out which pieces of the specification we consider "tier-1" or core, vs. going signal-by-signal.


E.g. Exemplars, Resources, Context are all *critical* to the project and aren't really called out in this model.

Reply:
The spec as a whole should be covered as part of the Spec SIG, included in the Tier-1 SIGs. I felt like we should list the signals here, but perhaps we should just remove this entire section? Stability for each signal is defined within the Spec SIG and it could list the individual signals (and other parts, like the ones you mentioned) in the tier-1 components?

Reply:
Given how the OTEP is right now, I think this doesn't belong here anymore, but this might still be important. The Collector part should be covered by this: https://github.com/open-telemetry/opentelemetry-collector/issues/8555 . What else do you think we need in the same area?

Reply:
@jsuereth, is there something to clarify around this, or can this be marked as resolved?

Reply:
I'm marking this as resolved, let me know if there's still something to address here.